poi/src/main/java/org/apache/poi/poifs/crypt/ChunkedCipherOutputStream.java

```diff
@@ -63,14 +64,14 @@ public abstract class ChunkedCipherOutputStream extends FilterOutputStream {
     private Cipher cipher;
     private boolean isClosed;
 
-    public ChunkedCipherOutputStream(DirectoryNode dir, int chunkSize) throws IOException, GeneralSecurityException {
+    public ChunkedCipherOutputStream(DirectoryNode dir, int chunkSize, TempFileCreationStrategy tmpStrategy) throws IOException, GeneralSecurityException {
```

Comment:
add back the old constructor for compat reasons

Reply:
Although truth to be told this class is marked as `@Internal`, so I guess when I was adjusting this, that was in my head.